src/core/client/stream/tabs/Comments/ReplyList/ReplyListCommentContainer.tsx

```diff
@@ -58,34 +63,54 @@ const ReplyListCommentContainer: FunctionComponent<Props> = ({
-          <CollapsableComment>
-            {({ collapsed, toggleCollapsed }) => (
-              <>
-                <DeletedTombstoneContainer comment={comment}>
-                  <CommentContainer
-                    viewer={viewer}
-                    comment={comment}
-                    story={story}
-                    collapsed={collapsed}
-                    settings={settings}
-                    indentLevel={indentLevel}
-                    localReply={localReply}
-                    disableReplies={disableReplies}
-                    showConversationLink={!!showConversationLink}
-                    toggleCollapsed={toggleCollapsed}
-                    showRemoveAnswered={showRemoveAnswered}
-                  />
-                </DeletedTombstoneContainer>
-                <div
-                  className={cn({
-                    [styles.hiddenReplies]: collapsed,
-                  })}
-                >
-                  {replyListElement}
-                </div>
-              </>
-            )}
-          </CollapsableComment>
+          {replyIsFlattened(flattenRepliesEnabled, indentLevel) ? (
+            <>
+              <DeletedTombstoneContainer comment={comment}>
+                <CommentContainer
+                  viewer={viewer}
+                  comment={comment}
+                  story={story}
+                  collapsed={false}
+                  settings={settings}
+                  indentLevel={indentLevel}
+                  localReply={localReply}
+                  disableReplies={disableReplies}
+                  showConversationLink={!!showConversationLink}
+                  showRemoveAnswered={showRemoveAnswered}
+                />
+              </DeletedTombstoneContainer>
+              {replyListElement}
+            </>
+          ) : (
+            <CollapsableComment>
+              {({ collapsed, toggleCollapsed }) => (
+                <>
+                  <DeletedTombstoneContainer comment={comment}>
+                    <CommentContainer
+                      viewer={viewer}
+                      comment={comment}
+                      story={story}
+                      collapsed={collapsed}
+                      settings={settings}
+                      indentLevel={indentLevel}
+                      localReply={localReply}
+                      disableReplies={disableReplies}
+                      showConversationLink={!!showConversationLink}
+                      toggleCollapsed={toggleCollapsed}
+                      showRemoveAnswered={showRemoveAnswered}
+                    />
+                  </DeletedTombstoneContainer>
+                  <div
+                    className={cn({
+                      [styles.hiddenReplies]: collapsed,
+                    })}
+                  >
+                    {replyListElement}
+                  </div>
+                </>
+              )}
+            </CollapsableComment>
+          )}
```

Comment:
What do you think about a way to avoid duplicating the jsx tags of line 61-88 (in the original file)? From my experience, this often leads to bugs in the future because someone applies a change to one and forgot about the duplicate.

Maybe something like this could do?
Thoughts?

```ts

          <CollapsableComment>
            {({ collapsed, toggleCollapsed }) => {
              const collapseEnabled = !replyIsFlattened(
                flattenRepliesEnabled,
                indentLevel
              );
              return (
                <>
                  <DeletedTombstoneContainer comment={comment}>
                    <CommentContainer
                      viewer={viewer}
                      comment={comment}
                      story={story}
                      collapsed={collapsed && collapseEnabled}
                      settings={settings}
                      indentLevel={indentLevel}
                      localReply={localReply}
                      disableReplies={disableReplies}
                      showConversationLink={!!showConversationLink}
                      toggleCollapsed={
                        collapseEnabled ? toggleCollapsed : undefined
                      }
                      showRemoveAnswered={showRemoveAnswered}
                    />
                  </DeletedTombstoneContainer>
                  <div
                    className={cn({
                      [styles.hiddenReplies]: collapsed && collapseEnabled,
                    })}
                  >
                    {replyListElement}
                  </div>
                </>
              );
            }}
          </CollapsableComment>
```